supertokens_python/recipe/session/utils.py

```diff
@@ -418,6 +418,17 @@ def validate_and_normalise_user_input(
     session_expired_status_code = (
         session_expired_status_code if session_expired_status_code is not None else 401
     )
+
+    invalid_claim_status_code = (
+        invalid_claim_status_code if invalid_claim_status_code is not None else 403
+    )
+
+    if session_expired_status_code == invalid_claim_status_code:
+        raise Exception(
+            "session_expired_status_code and invalid_claim_status_code cannot be the same "
+            f"({invalid_claim_status_code})"
```

Comment:
- What is the f"(..)" thing?
- Is the space after the word "same" intentional?

Reply:
- f-strings are a cleaner approach to format python strings. it allows you to do `f"hello {name_of_the_user}"` where `name_of_the_user` is a variable. 
- yes the space is intentional.